11_circular_csimeone/scr/plot_gages.R

```diff
@@ -0,0 +1,122 @@
+
+# library(tidyverse)       
+# library(readxl)
+library(tigris)
+# library(sf)
+# library(viridis)
+
+load("11_circular_csimeone/data_in/site_list.RData")
+
+# Read in metadata for plotting. 
+# gages 2 metadata can be found here: https://www.sciencebase.gov/catalog/item/59692a64e4b0d1f9f05fbd39
+loc_df <- read.csv("11_circular_csimeone/data_in/NWIS_data/gagesII_metadata.csv",header=TRUE,stringsAsFactors = FALSE) %>%
+  mutate(StaID = str_pad(STAID, width=8, side="left", pad = "0")) %>%
+  as_tibble() %>%
+  rename(site = StaID, latitude = LAT_GAGE, longitude = LNG_GAGE) %>%
+  select(c(site, latitude, longitude, HUC02)) %>%
+  filter(site %in% site_list)
+
+# # Basin shapefiles can be downloaded here. 
+# # https://apps.nationalmap.gov/downloader/#/
+# sh_14 <- sf::st_read("11_circular_csimeone/data_in/Shapefiles/HUC_14/WBDHU2.shp")
+# sh_f_14 <- fortify(sh_14)
+# sh_f_4326_14 <- st_transform(sh_f_14, 4326)[1]
+# 
+# sh_15 <- sf::st_read("11_circular_csimeone/data_in/Shapefiles/HUC_15/WBDHU2.shp")
+# sh_f_15 <- fortify(sh_15)
+# sh_f_4326_15 <- st_transform(sh_f_15, 4326)[1]
+# 
+# bbox <- c(left = -125, bottom = 25, right = -67, top = 50)
+# us <- c(left = -125, bottom = 25.75, right = -67, top = 49)
+# 
+# 
+# # site_map = ggmap(get_stamenmap(bbox, maptype = "terrain-background", zoom = 4))+
+# site_map <- ggmap(get_stamenmap(us, zoom = 5, maptype = "toner-lite")) + 
+#   geom_sf(data = sh_f_4326_14, inherit.aes = FALSE, fill = NA, size = 2) +
+#   geom_sf(data = sh_f_4326_15, inherit.aes = FALSE, fill = NA, size = 2) +
+#   geom_point(data = loc_df, aes(x = longitude, latitude),
+#              size = 1.5)+
+#   theme_bw() +
+#   scale_colour_gradientn(colours = rev(my.palette),
+#                          limits = c(-100, 100), 
+#                          oob = scales::squish) +
+#   labs(x = "Longitude", y = "Latitude")
+# site_map
+# 
+# site_map <- ggmap(data=us, map=us,aes(x=long, y=lat, map_id=region, group=group),
+#                   fill="#ffffff", color="#7f7f7f", size=0.5) + 
+#   geom_sf(data = sh_f_4326_14, inherit.aes = FALSE, fill = NA, size = 2) +
+#   geom_sf(data = sh_f_4326_15, inherit.aes = FALSE, fill = NA, size = 2) +
+#   geom_point(data = loc_df, aes(x = longitude, latitude),
+#              size = 1.5)+
+#   theme_bw() +
+#   scale_colour_gradientn(colours = rev(my.palette),
+#                          limits = c(-100, 100), 
+#                          oob = scales::squish) +
+#   labs(x = "Longitude", y = "Latitude")
+# site_map
+# 
+# us <- map_data("state")
+# us <- fortify(us, region="region")
+# 
+# p <- ggplot() + 
+#   geom_map(data=us, map=us,aes(x=long, y=lat, map_id=region, group=group),
+#            fill="#ffffff", color="#7f7f7f", size=0.5) + 
+#   geom_point(data = loc_df, aes(x=longitude, y=latitude)) + 
+#   coord_map("albers", lat0=39, lat1=45)+ 
+#   # coord_map('conic', lat0=30) +
+#   theme_map() +
+#   theme(legend.position = c(0.75, 0.3), plot.margin=unit(c(0,0,0,00),"mm")) 
+# p
```

Comment:
If none of this is used to make the final charts, delete it from the script

Reply:
Removed